Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataRecord.h

```diff
@@ -68,6 +68,9 @@ WK_EXTERN NSString * const WKWebsiteDataTypeSearchFieldRecentSearches WK_API_AVA
 /*! @constant WKWebsiteDataTypeMediaKeys MediaKeys storage */
 WK_EXTERN NSString * const WKWebsiteDataTypeMediaKeys WK_API_AVAILABLE(macos(WK_MAC_TBA), ios(WK_IOS_TBA));
 
+/*! @constant WKWebsiteDataTypeHashSalt Hash salt for deviceId */
```

Comment:
Ok, though after reading this comment I still have no idea what this clears.

Reply:
It's storage used for this type: https://developer.mozilla.org/en-US/docs/Web/API/MediaDeviceInfo/deviceId...